source/developers/concepts/hooks_extension_framework.rst

```diff
@@ -0,0 +1,136 @@
+Hooks Extension Framework
+#########################
+
+What is the Hooks Extension Framework?
+**************************************
+
+Based on the :doc:`edx-platform:concepts/extension_points`, this framework aims to extend the Open edX platform in a maintainable way without modifying its core. The main goal is to leverage the existing extension capabilities provided by the `Open edX Django plugins`_ architecture, allowing developers to implement new features to fit customer needs while reducing the need for core modifications and minimizing maintenance efforts.
```

Comment:
```suggestion
Based on the :doc:`edx-platform:concepts/extension_points`, this framework aims to extend the Open edX platform in a maintainable way without modifying its core. The main goal is to leverage the existing extension capabilities provided by the `Open edX Django plugins`_ architecture, allowing developers to implement new features while reducing the need for core modifications and minimizing maintenance efforts.
```

Reply:
@felipemontoya: would this be enough to address your concern?

Reply:
Done: https://github.com/openedx/docs.openedx.org/pull/599/commits/970122ddbb7c23d4c2f28ef9ee7a42e950d5e510